core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -1,16 +1,23 @@
 ---
+- name: Gather facts from repository
+  include_role:
+    name: repository
+    tasks_from: gather-facts
+
 - name: Check if Helm chart repo already exists
   shell: helm repo list | grep {{ helm_chart_repo_name }}
   args:
     executable: /bin/bash
   register: helm_repo_list
   failed_when: (helm_repo_list.rc != 0 and not 'no repositories' in helm_repo_list.stderr) 
                or helm_repo_list.rc > 1
+  when: inventory_hostname == target_repository_hostnames[0]
```

Comment:
Playing on all hosts but some tasks should be executed only on some of them, it is not clear I think, please comment.

Reply:
this PR is already covered by other one from sk4zuzu